ghost/oembed-service/lib/oembed-service.js

```diff
@@ -68,16 +67,7 @@ class OEmbed {
         this.config = config;
 
         /** @type {IExternalRequest} */
-        this.externalRequest = async (url, requestConfig) => {
-            if (this.isIpOrLocalhost(url)) {
-                return this.unknownProvider(url);
-            }
-            const response = await externalRequest(url, requestConfig);
-            if (this.isIpOrLocalhost(response.url)) {
-                return this.unknownProvider(url);
-            }
-            return response;
-        };
+        this.externalRequest = externalRequest; // changed this because this is already happening in externalRequest + we can't await because we lose .json() possibility on promise
```

Comment:
Gonna remove my own debug comment here 😊
```suggestion
        this.externalRequest = externalRequest;
```